cob_twist_controller/include/cob_twist_controller/constraints/constraint_ca_impl.h

```diff
@@ -389,22 +393,26 @@ void CollisionAvoidance<T_PARAMS, PRIO>::calcPredictionValue()
 
     if (params.frame_names.end() != str_it)
     {
+        unsigned int dof;
         if (this->constraint_params_.current_distances_.size() > 0)
         {
             uint32_t frame_number = (str_it - params.frame_names.begin()) + 1;  // segment nr not index represents frame number
             KDL::FrameVel frame_vel;
-
+            ROS_WARN_STREAM("frame_number" << frame_number);
```

Comment:
I suppose you used this warning for debugging purposes. Please remove such debugging outputs before PR